CRM/Utils/Date.php

```diff
@@ -258,13 +258,13 @@ public static function getAvailableInputFormats(bool $isShowTime): array {
   /**
    * Return abbreviated month names according to the locale.
    *
-   * @param bool $month
+   * @param bool|string $month
    *
    * @return array
    *   1-based array with abbreviated month names
    *
    */
-  public static function &getAbbrMonthNames($month = FALSE) {
+  public static function getAbbrMonthNames($month = FALSE): array {
```

Comment:
This type hint looks wrong - if `$month` is passed then output would be a string, right?

Reply:
yes! I have those functions!

Reply:
@colemanw fixed - I couldn't find any places in core where month is passed in so figured I would deprecate that parameter to work towards consistency